play-ebean/src/main/java/play/db/ebean/DefaultEbeanConfig.java

```diff
@@ -24,15 +24,24 @@
 /** Ebean server configuration. */
 @Singleton
 public class DefaultEbeanConfig implements EbeanConfig {
+  private final Boolean ddlGenerate;
 
   private final String defaultServer;
+
   private final Map<String, DatabaseConfig> serverConfigs;
 
-  public DefaultEbeanConfig(String defaultServer, Map<String, DatabaseConfig> serverConfigs) {
+  public DefaultEbeanConfig(
+      Boolean ddlGenerate, String defaultServer, Map<String, DatabaseConfig> serverConfigs) {
```

Comment:
I prefer to put the new parameter last:
```suggestion
  public DefaultEbeanConfig(
      String defaultServer, Map<String, DatabaseConfig> serverConfigs, Boolean generateEvolutionsScripts) {
```

Also, you can see the binary compatibility checks are failing, because you bascially removed a constructor. So you we have to keep that constructor to not break any user code. Please re-add it like:
```java
public DefaultEbeanConfig(String defaultServer, Map<String, DatabaseConfig> serverConfigs) {
  this(defaultServer, serverConfigs, true);
}
```
This will just call the new constructor, keeping generating the evolutions files.

Reply:
Initially I didn't quite understood what this was all about.